MdeModulePkg/Universal/PCD/Pei/Service.c

```diff
@@ -438,9 +439,28 @@ BuildPcdDatabase (
   //
   PeiPcdDbBinary = LocateExPcdBinary (FileHandle);
 
-  ASSERT (PeiPcdDbBinary != NULL);
+  if (PeiPcdDbBinary == NULL) {

```

Comment:
There's a few changes like this block still missing `MU_CHANGE` tags. Can you please look through one more time and check all the files?

Reply:
I added the missing MU_CHANGE tags.